documentation/frequently-asked-questions-for-azure-go-mgmt-sdk.md

```diff
@@ -0,0 +1,45 @@
+# Frequently Asked Questions for Azure Go Management SDK
+
+### 1. **JSON Unmarshal Error**
+
+When a user reports a bug with the description "JSON unmarshal error," follow these steps to resolve the issue:
+
+- **Step 1: Check the response body**
+  - Suggest the user open the logger to view the response body. This will allow us to inspect the details returned by the API and determine if the issue is related to the SDK.
+  
+  Example:  
+  If the SDK's unmarshaling result differs from the API response, the issue may lie with the SDK's handling of the data.
+
+- **Step 2: Verify the SDK version**
+  - Check the reported SDK version. If it is not the latest, suggest the user try the latest version locally. If the latest version resolves the issue, recommend upgrading to the latest version.
+  
+  Example:  
+  If the issue is fixed with the latest version, suggest upgrading to that version.
+
+- **Step 3: Add 'service-attention' label**
+  - If the issue persists, label the ticket with `service-attention`.
+
+---
+
+### 2. **Add 'Service-Attention' Label**
+
+When a user reports issues related to product experience or functionality of the service, add the `service-attention` label. For example:
```

Comment:
Is this still true? I have also been to that customers should open Azure support tickets in such cases.

CC @sandeep-sen

Reply:
Agreed, we should be closing these issues and having customers open support tickets on the service.